deps/zlib.BUILD.bazel

```diff
@@ -162,6 +162,7 @@ pkg_install(
         ":lib_files",
         ":license_files",
     ],
+    destdir_flag = "@@//:install_dir",
```

Comment:
Would it be possible to have the flag declared as a default on `rules_pkg`'s end? In order to avoid having to add this to all the invocations of `pkg_install`

Reply:
that's fair, would it be possible to provide a default flag value when loading the module then?